cmd/globals.go

```diff
@@ -96,6 +99,12 @@ var (
 	// IsSSL indicates if the server is configured with SSL.
 	globalIsSSL bool
 
+	globalTLSCertificate *tls.Certificate
+
+	globalHTTPServer        *miniohttp.Server
+	globalHTTPServerErrorCh = make(chan error)
+	globalOSSignalCh        = make(chan os.Signal, 1)
```

Comment:
Reading of this channel is done only by `handleSignals()` and once value on the channel received, it calls `os.Exit()`.  I think closing channel is not useful.  Moreover there is a chance of panic `write on closed channel` by writer `signal.Notify()`